### PR TITLE
Publishing to npm

### PR DESCRIPTION
**Pull request by ryan-blunden**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: publish
+
+on:
+  release:
+    types: [published]
+  workflow_run:
```
Comment (nmanoogian) [inline .github/workflows/publish.yml]: Good for testing but we'll want to remove this before merge, yes? Or am I misunderstanding?

File: package.json [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
   "repository": {
     "type": "git",
-    "url": "git://github.com/doppler-university/gitops-secrets.git"
+    "url": "https://github.com/doppler-university/gitops-secrets.git"
   },
-  "version": "0.0.1-DEV",
   "homepage": "https://github.com/doppler-university/gitops-secrets#readme",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/axios/axios/issues"
```
Comment (nmanoogian) [inline package.json]: We'll probably want to change this to https://github.com/DopplerUniversity/gitops-secrets-nodejs/issues yeah?

File: package.json [hunk 3]
```diff
@@ -1,13 +1,17 @@
 {
   "name": "gitops-secrets",
+  "version": "0.0.1",
   "author": "Ryan Blunden <ryan.blunden@doppler.com>",
   "description": "GitOps encrypted secrets workflow",
   "repository": {
     "type": "git",
-    "url": "git://github.com/doppler-university/gitops-secrets.git"
+    "url": "https://github.com/doppler-university/gitops-secrets.git"
```
Comment (nmanoogian) [inline package.json]: It's actually https://github.com/DopplerUniversity/gitops-secrets-nodejs, right?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+```sh
+npm run prettier
+npm run lint
+```
+
+## Commit Messages
+
+Commit messages should be verb based, using the following pattern:
+
+- `Fixing ...`
```
Comment (nmanoogian) [inline CONTRIBUTING.md]: nit: Could we phrase these in the present simple tense? E.g. `Fixes`, `Adds`, `Updates`, etc.

That will keep this project consistent with our other open-source repos

**Review by nmanoogian — APPROVED** [review]

File: .github/workflows/publish.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  publish:
+    if: github.ref == 'refs/heads/main'
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
```
Comment (Piccirello) [inline .github/workflows/publish.yml:16]: Can you use Node 16 here?

Comment (ryan-blunden) [reply .github/workflows/publish.yml:16]: See https://github.com/DopplerUniversity/gitops-secrets-nodejs/pull/14

**Pull request merged**